src/main/java/plugily/projects/thebridge/Main.java

```diff
@@ -91,17 +84,26 @@ public void addKits() {
     long start = System.currentTimeMillis();
     getDebugger().debug("Adding kits...");
     addFileName("kits");
-    Class<?>[] classKitNames = new Class[]{BridgeKit.class, KnightKit.class, LightTankKit.class, ArcherKit.class, HealerKit.class,
-      MediumTankKit.class, TerminatorKit.class, HardcoreKit.class, PremiumHardcoreKit.class, NakedKit.class, HeavyTankKit.class};
-    for(Class<?> kitClass : classKitNames) {
-      try {
-        kitClass.getDeclaredConstructor().newInstance();
-      } catch(Exception e) {
-        getLogger().log(Level.SEVERE, "Fatal error while registering existing game kit! Report this error to the developer!");
-        getLogger().log(Level.SEVERE, "Cause: " + e.getMessage() + " (kitClass " + kitClass.getName() + ")");
-        e.printStackTrace();
-      }
+    
+    List<String> optionalConfigurations = new ArrayList<>();
+    optionalConfigurations.add("bow-cooldown");
+
+    FileConfiguration kitsConfig = ConfigUtils.getConfig(this, "kits");
+
+    if (!Objects.equals(kitsConfig.getString("Do-Not-Edit.File-Version"), "2")) {
+      getLogger().log(Level.SEVERE, "Your kits.yml config is outdated. Please update it.");
+      getLogger().log(Level.SEVERE, "Cause: File-Version is not 2");
+      return;
+    }
+    if (!Objects.equals(kitsConfig.getString("Do-Not-Edit.Core-Version"), "1")) {
+      getLogger().log(Level.SEVERE, "Your kits.yml config is outdated. Please update it.");
+      getLogger().log(Level.SEVERE, "Cause: Core-Version is not 1");
+      return;
```

Comment:
Don't think we really need, we just update with our updated to latest so it should work without it ;)
Remove the lines